app/src/test/java/org/wikipedia/login/UserExtendedInfoClientTest.kt

```diff
@@ -9,27 +10,10 @@ class UserExtendedInfoClientTest : MockRetrofitTest() {
     fun testRequestSuccess() {
         enqueueFromFile("user_extended_info.json")
         val id = 24531888
-        apiService.userInfo.test().await()
-            .assertComplete().assertNoErrors()
-            .assertValue {
-                it.query?.userInfo?.id == id &&
-                        it.query?.getUserResponse("USER")?.name == "USER"
-            }
-    }
-
-    @Test
-    @Throws(Throwable::class)
-    fun testRequestResponse404() {
-        enqueue404()
```

Comment:
I removed the rest of these tests because I no longer understand what they're testing. Are they testing if Retrofit works correctly? That's already done in Retrofit's own test suite. Are they testing if the Java language works correctly? I'm not sure that's within the scope of what we should worry about.